openpype/hosts/maya/plugins/publish/submit_maya_muster.py

```diff
@@ -52,6 +52,11 @@ def _get_script():
     return module_path
 
 
+def get_renderer(layer):
```

Comment:
It might not be used by any other plugin now, but I'd say this is fine staying in `lib`.

Reply:
Should be expose in the api init then we expect to be used externally?

Reply:
I would just keep it around in lib for if another plug-in might need it. No need to expose it further than this I think.

Reply:
I'm more thinking that exposing it in api ini would indicate for future reference that it needs to be kept for external use?

Reply:
ok, kept it in